content/Interrupts.md

```diff
@@ -56,7 +56,7 @@ interrupt "waits" until both IME and IE allow its execution.
 
 ### Interrupt Execution
 
-When an interrupt is executed, the corresponding bit in the IF
+Just before an interrupt is executed, the corresponding bit in the IF
```

Comment:
This doesn't make much sense given that this whole description includes the calling part. I think it would be better to describe that this is the process known as "interrupt handling", and break the separate steps into a numbered list.

Reply:
Fixed